### PR TITLE
feat: add alertSettings.parallelRunFailureTreshold to CLI [sc-19200]

### PR DESCRIPTION
**Pull request by maxigimenez**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

## Affected Components
* [x] CLI
* [ ] Create CLI
* [ ] Test
* [ ] Docs
* [ ] Examples
* [ ] Other

## Notes for the Reviewer
Add new alert settings options:

```
alertEscalationPolicy: {
    parallelRunFailureThreshold: {
      enabled: true,
      percentage: 100,
    }
  }
```

### CODE REVIEW

File: packages/cli/src/constructs/alert-escalation-policy.ts [hunk 1]
```diff
@@ -9,6 +9,11 @@ export type Reminders = {
   interval?: number
 }
 
+export type ParallelRunFailureThreshold = {
+  enabled?: boolean,
+  percentage?: number,
```
[the remark this answers is not in the document]
Comment (shiini2) [reply packages/cli/src/constructs/alert-escalation-policy.ts:14]: If they're missing the BE schema should apply default values

[the remark this answers is not in the document]
Comment (maxigimenez) [reply packages/cli/src/constructs/alert-escalation-policy.ts:14]: That should be the case yes, it fallback always to the schema defaults

[the remark this answers is not in the document]
Comment (maxigimenez) [reply packages/cli/src/constructs/alert-escalation-policy.ts:14]: I've follow the same pattern as the other alertSettings props in this file

[the remark this answers is not in the document]
Comment (Antoine-C) [reply packages/cli/src/constructs/alert-escalation-policy.ts:14]: Gotcha, it's weird the `enabled` is optional and defaults to something though but anyway LGTM

**Review by Antoine-C — APPROVED** [review]

**Pull request merged**